orchestrate/dags/yaml_sample_slack_dag.yml

```diff
@@ -0,0 +1,41 @@
+yaml_sample_slack_dag:
+  description: "Sample yaml dag dbt run"
+  schedule_interval: "0 0 1 */12 *"
+  tags:
+    - version_5
+  catchup: false
+
+  default_args:
+    start_date: 2023-01-01
+    owner: airflow
+    # Replace with the email of the recipient for failures
+    email: gomezn@datacoves.com
+    email_on_failure: true
+  custom_callbacks:
+      on_success_callback:
+          module: callbacks.slack_messages
+          callable: inform_success
+          args:
+              - connection_id: SLACK_INTEGRATION
+              # - message: Custom YML success message
+              # - color: 0000FF
+      on_failure_callback:
+          module: callbacks.slack_messages
+          callable: inform_failure
+          args:
+              - connection_id: SLACK_INTEGRATION
```

Comment:
can you follow the same conventions I did in the Teams dags?
like
              - connection_id: DATACOVES_MS_TEAMS

Reply:
Updated